.env.example

```diff
@@ -21,3 +21,5 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 
 AWS_PINPOINT_REGION=us-west-2
+
+LOAD_TEST_DOMAIN=https://api.staging.notification.cdssandbox.xyz
```

Comment:
This would not be necessary I think.. as the `.env` file that we read are actually in the *locust* folder and not at the root. This won't get used by the main Flask application, right?

Reply:
Yes you are right.. It is defined there as an example.

Reply:
After this merges we should have a PR that reorgs/refactors the code a bit so that the smoke tests and locust tests can share env vars (ie templates). But that should definitely not be done in this PR.